org.gnome.Screenshot.json

```diff
@@ -0,0 +1,72 @@
+{
+  "app-id": "org.gnome.Screenshot",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
+  "sdk": "org.gnome.Sdk",
+  "command": "gnome-screenshot-wrapper",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--talk-name=org.gnome.Shell.Screenshot",
+    "--filesystem=xdg-run/dconf",
+    "--filesystem=~/.config/dconf:ro",
+    "--talk-name=ca.desrt.dconf",
+    "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+    "--filesystem=home"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/share/pkgconfig",
+    "/share/aclocal",
+    "/man",
+    "/share/man",
+    "/share/gtk-doc",
+    "/share/vala",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "libcanberra-gtk3",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://0pointer.de/lennart/projects/libcanberra/libcanberra-0.30.tar.xz",
+          "sha256": "c2b671e67e0c288a69fc33dc1b6f1b534d07882c2aceed37004bf48c601afa72"
+        }
+      ]
+    },
+    {
+      "name": "wrapper",
```

Comment:
Why have it as separate module?

Reply:
Where I can put it then? I can have it as a source for the gnome-screenshot module but how can I install it?

Reply:
a source in the gnome-screenshot one and a post-install module rather build-commands.